Typing error when using library

In my project, typescript failed to compile when using this dependency of `jest-mock-axios`
![image](https://user-images.githubusercontent.com/9356633/37953559-6ec71684-31cd-11e8-87b5-1d5dad714929.png)


The error comes from this function typing
```
type AnyFunction = (any)=>any;
```

Error message
```
/Users/minheq/Documents/Repositories/app-boilerplate/node_modules/jest-mock-promise/dist/jest-mock-promise-types.d.ts
(6,29): error TS7006: Parameter 'any' implicitly has an 'any' type.
```

--- a/lib/jest-mock-promise-types.ts
+++ b/lib/jest-mock-promise-types.ts
@@ -4,7 +4,7 @@ enum PromiseState {
     rejected
  }
 
- type AnyFunction = (any)=>any;
+ type AnyFunction = (...args: any[])=>any;
 
  type HandlerType = {
      then?:AnyFunction,

